docs/topics/testing/tools.txt

```diff
@@ -153,25 +158,26 @@ Use the ``django.test.Client`` class to make requests.
 
             /customers/details/?name=fred&age=7
 
-        The ``extra`` keyword arguments parameter can be used to specify
-        headers to be sent in the request. For example::
+        The ``headers`` parameter can be used to specify headers to be sent in
+        the request. For example::
 
             >>> c = Client()
             >>> c.get('/customers/details/', {'name': 'fred', 'age': 7},
-            ...       HTTP_ACCEPT='application/json')
+            ...       headers={'Accept': 'application/json'})
 
         ...will send the HTTP header ``HTTP_ACCEPT`` to the details view, which
         is a good way to test code paths that use the
         :meth:`django.http.HttpRequest.accepts()` method.
 
-        .. admonition:: CGI specification
+        Arbitrary keyword arguments set WSGI `environ variables`_. For example,
+        headers can be set with the ``HTTP_`` prefix::
```

Comment:
As above, showing `SCRIPT_NAME` instead of `HTTP_*`:

```suggestion
        Arbitrary keyword arguments set WSGI `environ variables`_. For example,
        headers to set the script name::
```